Maestro/src/Microsoft.DotNet.Maestro.WebApi/Web.config

```diff
@@ -8,7 +8,7 @@
     <add key="WebhookSecretToken" value="" />
     <add key="KeyVaultClientId" value="" />
     <add key="KeyVaultSecret" value="" />
-    <add key="dotnet.visualstudio.com-KeyVaultSecretName" value="" />
+    <add key="dnceng.visualstudio.com-KeyVaultSecretName" value="" />
```

Comment:
This won't take effect until deployment (which is manual), so I think you (or FR, or someone) will need to deploy and update the value in the app service. https://github.com/dotnet/core-eng/blob/master/Documentation/Maestro/deploy.md

Reply:
Not sure. I'd expect an auth error when it tries to trigger the job (or maybe it would just hit the 404).